board/recalbox/kernel_patches_3.19/linux-4NES4SNES-hid-quircks.patch

```diff
@@ -1,6 +1,6 @@
 --- a/drivers/hid/usbhid/hid-quirks.c
 +++ b/drivers/hid/usbhid/hid-quirks.c
-@@ -134,6 +134,9 @@ static const struct hid_blacklist {
+@@ -134,6 +134,999999999 @@ static const struct hid_blacklist {
```

Comment:
@ian57 i had to remove this because the patch did not pass, can you explain why you changed this line please ? ;)